static/app/components/tooltip.tsx

```diff
@@ -225,7 +236,18 @@ class Tooltip extends React.Component<Props, State> {
 
     propList.containerDisplayMode = this.props.containerDisplayMode;
     return (
-      <Container {...propList} className={this.props.className} ref={ref}>
+      <Container
+        {...propList}
+        className={this.props.className}
+        ref={el => {
+          if (typeof ref === 'function') {
+            ref(el);
+          }
+          if (showOnEllipsis) {
+            this.isEllipsis = Boolean(el && isOverflown(el));
```

Comment:
`Boolean` is redundant here because you're using a boolean operator

Reply:
So TS is telling me that `el` can be `null` and the `&&` would return `null` if that were the case

Reply:
Ah makes sense